lib/ansible/modules/windows/win_file.py

```diff
@@ -28,32 +28,53 @@
 ---
 module: win_file
 version_added: "1.9.2"
-short_description: Creates, touches or removes files or directories.
+short_description: Sets attributes of files
 description:
-     - Creates (empty) files, updates file modification stamps of existing files,
-       and can create or remove directories.
-     - Unlike M(file), does not modify ownership, permissions or manipulate links.
+  - Sets attributes of files, symlinks, hard links, junction points and directory.
+  - Can create files, symlinks, hard links, junction points, directories and files.
+  - Unlike M(file), does not modify ownership or permissions.
 notes:
-    - See also M(win_copy), M(win_template), M(copy), M(template), M(assemble)
-requirements: [ ]
+  - See also M(win_copy), M(win_template), M(copy), M(template), M(assemble)
 author: "Jon Hawkesworth (@jhawkesworth)"
 options:
+  force:
+    description:
+      - This flag indicates whether to force the creation of a link in the
+        following cases.
+      - If state=C(link) and C(path) exists and is not a symbolic link, the
+        C(path) will be deleted and set as a new symbolic link.
+      - If state=C(junction) and C(path) exists and is not a junction point,
+        the C(path) will be deleted and set as a new junction point.
+      - If state=C(hard) and C(path) exists and is not a hard link of C(src),
+        the C(path) will be deleted and set as a new hard link.
+    choices: [true, false]
```

Comment:
Is there a link or document that states this, it's a bit frustrating trying to keep up with these standards sometimes.